sdks/python/container/boot.go

```diff
@@ -336,7 +342,11 @@ func setupVenv(ctx context.Context, logger *tools.Logger, baseDir, workerId stri
 	if err := os.MkdirAll(dir, 0750); err != nil {
 		return "", fmt.Errorf("failed to create Python venv directory: %s", err)
 	}
-	if err := execx.Execute("python", "-m", "venv", "--system-site-packages", dir); err != nil {
+	pythonVersion, err := expansionx.GetPythonVersion()
+	if err != nil {
+		return "", fmt.Errorf("python installation not found: %v", err)
```

Comment:
should we return `err` without modifications here this message is already included?

Reply:
done